risc0/zkvm/platform/src/syscall.rs

```diff
@@ -72,6 +70,8 @@ pub mod reg_abi {
     pub const REG_T4: usize = 29; // temporary
     pub const REG_T5: usize = 30; // temporary
     pub const REG_T6: usize = 31; // temporary
+    pub const REG_PC: usize = 32; // program counter
```

Comment:
Does this imply that we'll remove pc from the `MachineContext` struct and use this instead? I'm asking because it (positively) affects the gdb support that I'm working on

Reply:
My plan is to have the PC be recorded in system memory just before pausing a session. We need to record it for the next sessions so that:
1. we can verify that the PC was in a particular position
2. we can resume at the last PC position

The `MachineContext` will probably change a lot, but I don't expect to have all that working for awhile yet.